website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,24 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
 
+Organization owners can choose between Non-Aggregated (Default) and Aggregated status checks. This setting determines whether detailed information and links are accessed directly from the VCS Provider or Terraform Cloud.
+It also determines the quantity of status checks directly sent to the VCS Provider in response to actions such as Pull/Merge requests.
+
+##### Aggregated Status Checks
+
+Aggregated status checks offer a streamlined experience to customers with "monorepo" repositories, containing configuration for a large number of workspaces.
+
+When enabled, Terraform Cloud will send one VCS status check for all runs triggered by a VCS event. If multiple workspaces are tied to a shared repository, status checks for these workspaces will be aggregated into one summary. This summary is unique to the workspace's organization and VCS client connection.
+
+The aggregated status check will allow users to access additional information in Terraform Cloud by accessing the provided "details" link. This link navigates users to a page in Terraform Cloud that offers a consolidated presentation across multiple Workspaces, highlighting details such as resource changes and possible issues that require user's attention.
+
+##### Non-aggregated Status Checks
+
+Default behavior where the VCS provider receives a status for each triggered Workspace and related Run stages (e.g. Plan, Policy Checks, Cost Estimation, Run Tasks, etc.) in response to a VCS event.
+
+###### Send passing commit statuses
+-> **Note:** Organization owners can only enable **Send passing commit statuses** option if **Aggregated status checks** is disabled.
```

Comment:
Do we also need a corresponding note above?  So folks know they can't enable aggregated status checks unless they have **Send passing commit statuses** disabled?
```suggestion
-> **Note:** Organization owners can only enable the **Send passing commit statuses** setting if the **Aggregated status checks** setting is disabled.
```

Reply:
I don't think we do since the settings page experience makes this option visible only if it's disabled